core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -463,6 +463,12 @@ class TimeSeriesShard(val ref: DatasetRef,
 
   private[memstore] val addPartitionsDisabled = AtomicBoolean(false)
 
+  private[memstore] val housekeepingSched = Scheduler.computation(
+    name = "housekeeping",
+    reporter = UncaughtExceptionReporter(logger.error("Uncaught Exception in Housekeeping Scheduler", _)))
+
+  private var houseKeepingFuture: CancelableFuture[Unit] = initHousekeeping()
```

Comment:
Will probably rename anything "housekeeping-interval"-related to "reader-refresh-interval"